src/org/exist/util/serializer/AbstractSerializer.java

```diff
@@ -30,7 +30,7 @@
         new HTML5Writer()
     };
 
-    private final static Properties defaultProperties = new Properties();
+    protected final static Properties defaultProperties = new Properties();
```

Comment:
Okay, I think that maybe the following change from:

```java
if (properties == null) {
    outputProperties = defaultProperties;
```

to:

```java
if (properties == null) {
    outputProperties = new Properties(defaultProperties);
```

might be a good idea to prevent accidental modification in AdaptiveWriter or other places.